service-rpc/src/main/java/org/apache/iotdb/rpc/IoTDBRpcDataSet.java

```diff
@@ -343,9 +352,11 @@ public long getLong(String columnName) throws StatementExecutionException {
     }
     int index = columnOrdinalMap.get(columnName) - START_INDEX;
     if (!isNull(index, rowsIndex - 1)) {
+      lastReadWasNull = false;
       return BytesUtils.bytesToLong(values[index]);
     } else {
-      throw new StatementExecutionException(String.format(VALUE_IS_NULL, columnName));
```

Comment:
Maybe throw an exception is better? which can help users find bugs in client

Reply:
The reason why I remove the exception here is that it was not consistent with JDBC specification as we discussed in [IOTDB-912](https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-912?filter=allopenissues).

Reply:
ok thanks!